tools/osv-scanner/main.go

```diff
@@ -4,13 +4,16 @@ import (
 	"bufio"
 	"bytes"
 	"errors"
+	"golang.org/x/crypto/ssh/terminal"
```

Comment:
This should go into the third party block from line 14 too, since it's not a standard library.